test/integration/commands/pluck_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Integration.Commands.PluckTest do
+  use Cog.AdapterCase, adapter: "test"
+
+  setup do
+    user = user("lucky", first_name: "Shamaus", last_name: "McLucky")
+    |> with_chat_handle_for("test")
+
+    {:ok, %{user: user}}
+  end
+
+  test "with no field option", %{user: user} do
```

Comment:
I like the suggestion @christophermaier. Another approach that I've taken with some of the tests is to generate the map that I expect from the command's output and render the commands template with it. https://github.com/operable/cog/blob/master/test/integration/commands/which_test.exs#L12-L20